tasks/collect-cosign-params/collect-cosign-params.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-cosign-params
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task that collects cosign options from the data file
+  params:
+    - name: dataPath
+      type: string
+      description: Path to the JSON string of the merged data containing the configuration options to use
+  workspaces:
+    - name: data
+      description: Workspace to read and save files
+  results:
+    - name: cosign-secret
+      type: string
+      description: Name of secret which contains AWS_REGION, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and SIGN_KEY
+  steps:
+    - name: collect-params
+      image: quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393
+      script: |
+        #!/usr/bin/env sh
+        set -x
+
+        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        if [ ! -f "${DATA_FILE}" ] ; then
+            echo "No valid data file was provided."
+            exit 1
+        fi
+
+        if [ $(jq '.sign | has("cosign-secret")' $DATA_FILE) == false ] ; then
+            echo "No secret name provided via 'sign.cosign-secret' key in Data."
```

Comment:
I see we already use `.sign.configMapName` for something else: https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/rh-sign-image/rh-sign-image.yaml#L65

So maybe use `.sign.cosignSecretName` to be consistent with that naming?

Reply:
Renamed it.

Reply:
Thanks.